projects/lf-documentation/src/app/getting-started-cdn-example/getting-started-cdn-example.component.html

```diff
@@ -0,0 +1,154 @@
+<h3 id="cdn-example"> Example: <code>lf-checklist</code> In an Angular Project Via CDN</h3>
+<p>In the example below, the implementation comes from the CDN and the types is provided
+  in the <code>@laserfiche/types-lf-ui-components</code> NPM package.</p>
+<ol>
+  <li>
+    <p  class="documentation-text">
+      Add the following script tags to <code>index.html</code>
+    </p>
+    <pre class="code-container">
+<span class="code-html-tagname"> &lt;script </span><span class="code-html-attribute-name">src</span><span>=</span><span class="code-html-attribute-value"
+>"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/laserfiche-ui-components.js"</span><span class="code-html-attribute-name"
+> defer</span><span class="code-html-tagname">&gt;&lt;/script&gt;</span>
+<span class="code-html-tagname"> &lt;link </span><span class="code-html-attribute-name">href</span><span>=</span><span class="code-html-attribute-value"
+>"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/lf-laserfiche-lite.css"</span><span class="code-html-attribute-name"> rel</span><span
+>=</span><span class="code-html-attribute-value">"stylesheet" </span><span class="code-html-tagname">/&gt;</span>
+<span class="code-html-tagname"> &lt;link </span><span class="code-html-attribute-name">href</span><span>=</span><span class="code-html-attribute-value"
+  >"https://unpkg.com/@laserfiche/lf-ui-components@NPM_VERSION/cdn/indigo-pink.css"</span><span class="code-html-attribute-name"> rel</span><span
+  >=</span><span class="code-html-attribute-value">"stylesheet" </span><span class="code-html-tagname">/&gt;</span>
+</pre>
+</li>
+    <li>
+      Install the types definition for the custom elements.
+        <pre class="code-container"> npm install @laserfiche/types-lf-ui-components@NPM_VERSION </pre>
+    </li>
+  <li>
+    <p>
+      Inside <code>./app.module.ts</code>, add <code>CUSTOM_ELEMENTS_SCHEMA</code> to @NgModule.schemas. This
+      enables custom elements in your Angular application.
```

Comment:
This allows you to use custom non-Angular components in your Angular application